algoliasearch-core/src/test/java/com/algolia/search/integration/recommendation/RecommendationTest.java

```diff
@@ -25,6 +22,16 @@ protected RecommendationTest(RecommendationClient recommendationClient) {
 
   @Test
   void testRecommendationClient() {
+    SetStrategyRequest request =
```

Comment:
I think this test can be a little bit better.

Maybe you can do like this: https://github.com/algolia/algoliasearch-client-javascript/blob/3991e821b46c860e1badda6dbfecadd3d73701d3/packages/client-recommendation/src/__tests__/features/personalization-strategy.test.ts#L48.

Reply:
Got it! Will change it so that the payload is also checked for equality upon reception.